src/components/calendar/calendar.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const Calendar = () => (
+  <>
```

Comment:
Don't need a fragment here if everything is under a single div.

Fragments are for easy array returning

```javascript
<>
  <div>1</div>
  <div>2</div> 
</>
```

Reply:
AHHH ok, didnt know that